code/modules/mob/living/carbon/human/life/handle_chemicals_in_body.dm

```diff
@@ -42,7 +42,7 @@
 		if(species.flags & IS_PLANT)
 			if(nutrition > 500)
 				nutrition = 500
-			if(light_amount >= 3 && !reagents.has_reagent(HYPERZINE)) //If there's enough light, and you do not have hyperzine in body, heal
+			if(light_amount >= 3 && !reagents.has_reagent(HYPERZINE) && !reagents.has_reagent(PLANTBGONE)) //If there's enough light, you do not have hyperzine in body, and you don't have plant-b-gone inside you, heal
```

Comment:
please instead use `!reagents.has_any_reagents(list(HYPERZINE,PLANTBGONE))` - this is shorter, more readable, and more flexible

Reply:
done